discovery-infra/test_infra/helper_classes/cluster_host.py

```diff
@@ -0,0 +1,50 @@
+import json
+from typing import List
+
+from assisted_service_client import Inventory, Host, Interface
+
+DEFAULT_HOSTNAME = "localhost"
+
+
+class ClusterHost:
+    def __init__(self, host_model: Host):
+        self.__hostModel = host_model
```

Comment:
why the double underscore?
I don't know this convention

Reply:
also having capital letters in member names

Reply:
underscore means protected member and double underscore means private members. It can be protected but i think no one should inherit from this class.
not sure that the python convention is to capitalize the private/protected members.

Reply:
didn't know about difference of protected/private
about capitalization, that's exactly my point. please change to ``__host_model`` instead of ``__hostModel``